inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/StreamSourceEntityMapper.java

```diff
@@ -190,6 +190,11 @@ void updateStatusByIds(@Param("idList") List<Integer> idList, @Param("status") I
      */
     void updateStatusByDeleted();
 
+    /**
+     * Logic delete the sub source by modifiy time
+     */
+    void logicalDeleteByTimeout(@Param("beforeDay") Integer beforeDays);
+
```

Comment:
Please clarify the meaning of parameter `beforeDays`

Reply:
Fixed.